src/MyWool.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Heading } from "@chakra-ui/react"
+import { StatGroup } from "@chakra-ui/stat"
+import { useWallet } from "./context/wallet";
+import { useContract } from './hooks/useContract';
+import { WOOL_CONTRACT_ADDRESS } from './constants';
+import woolAbi from './constants/woolAbi.json';
+import { useEffect, useState } from "react";
+import { BigNumber } from "@ethersproject/bignumber";
+import StatItem from './StatItem';
+import useWoolQuery from './hooks/useWoolQuery';
+import { Ticker } from "./types"
+
+
+type Props = {
+	stats?: Ticker
+}
+
+function useMyWool() {
+  const { currentAccount } = useWallet();
+  const contract = useContract(WOOL_CONTRACT_ADDRESS, woolAbi);
+  const { data: woolData } = useWoolQuery()
+  const [woolAmount, setWoolAmount] = useState<number | null>(null);
+  console.log("data", woolData);
+
+  useEffect(() => {
+    const getWool = async () => {
+      if (currentAccount && contract) {
+        const gweiResult = await contract.balanceOf(currentAccount) as BigNumber;
+        const amount = gweiResult.toNumber() * Math.pow(10, -18);
```

Comment:
The `toNumber` is throwing an error for me because it's a BigNumber
If the token has 18 decimals it looks like we can use `formatEther` util from ethers + parseFloat to get the correct conversion
```ts
const amount = parseFloat(utils.formatEther(gweiResult))
```

Reply:
Good find! Can you try now?

Reply:
Yep, works well now